tests/cancel_task.yml

```diff
@@ -1,6 +1,9 @@
 name: Cancel Tes Task Job
 description: Job to cancel a TES Task
 service: TES
+compatible_versions:
+  - v1.0
+  - v1.1
```

Comment:
Would be better to use versions that follow [SemVer](http://semver.org/), i.e., no leading `v`. Also may include the patch version, I think. So in this case `1.0.0` and `1.1.0` (these are also used in the TES specs themselves). Or you could have define rules how to specify flexible matching, e.g., `1.1` would match _all_ patch versions of `1.1`, and `1` would match _all_ minor versions of `1`, including `1.0.0` and `1.1.0`.